controllers/volumereplicationgroup_controller.go

```diff
@@ -790,11 +815,8 @@ func (v *VRGInstance) deleteVRGHandleMode() bool {
 		asyncModeRequeue = v.reconcileVRsForDeletion()
 	}
 
-	// for now nothing to do for MetroDR. VRG does not create VolRep or
-	// any other resource for MetroDR. Hence set syncModeRequeue to false
-	// indicating metroDR does not require a requeue.
 	if v.instance.Spec.Sync.Mode == ramendrv1alpha1.SyncModeEnabled {
```

Comment:
Why don't you just drop the IFs?  Have one call to `v.reconcileVRsForDeletion`.  Make simple and change it to one line function.  In the future if we need to handle more modes (I don't know what that is) we can add easily add them.